### ISSUE TITLE
Fatal error in ../deps/v8/src/api.cc, line 1051

### ISSUE THREAD

**Issue by retorquere** [jogleasonjr/react-bootstrap-starter#3]
I've cloned the repo and ran `npm install`, but when I run `gulp` I get

```
#
# Fatal error in ../deps/v8/src/api.cc, line 1051
# Check failed: !value_obj->IsJSReceiver() || value_obj->IsTemplateInfo().
#

==== C stack trace ===============================

    0   node                                0x00000001009569bd v8::base::debug::StackTrace::StackTrace() + 19
    1   node                                0x00000001009556d5 V8_Fatal + 233
    2   node                                0x0000000100144d10 v8::Template::SetAccessorProperty(v8::Local<v8::Name>, v8::Local<v8::FunctionTemplate>, v8::Local<v8::FunctionTemplate>, v8::PropertyAttribute, v8::AccessControl) + 0
    3   fse.node                            0x00000001027c3c33 fse::FSEvents::Initialize(v8::Local<v8::Object>) + 197
    4   node                                0x0000000100791425 node::DLOpen(v8::FunctionCallbackInfo<v8::Value> const&) + 855
    5   node                                0x000000010016a096 v8::internal::FunctionCallbackArguments::Call(void (*)(v8::FunctionCallbackInfo<v8::Value> const&)) + 378
    6   node                                0x00000001001ab0ca v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) + 915
    7   node                                0x00000001001aa6a7 v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) + 281
    8   ???                                 0x00002977439063a7 0x0 + 45592211383207
Illegal instruction: 4
```

**Comment by jogleasonjr** [comment]
Hey @retorquere, thanks for letting me know. I've updated the react dependencies which should solve any build problems. Mind giving it another try? Thanks!

**Comment by retorquere** [comment]
I still get 

```
$ gulp


#
# Fatal error in ../deps/v8/src/api.cc, line 1051
# Check failed: !value_obj->IsJSReceiver() || value_obj->IsTemplateInfo().
#

==== C stack trace ===============================

    0   node                                0x00000001009569c1 v8::base::debug::StackTrace::StackTrace() + 19
    1   node                                0x00000001009556d9 V8_Fatal + 233
    2   node                                0x0000000100144b10 v8::Template::SetAccessorProperty(v8::Local<v8::Name>, v8::Local<v8::FunctionTemplate>, v8::Local<v8::FunctionTemplate>, v8::PropertyAttribute, v8::AccessControl) + 0
    3   fse.node                            0x00000001021efbb1 fse::FSEvents::Initialize(v8::Local<v8::Object>) + 197
    4   node                                0x0000000100791225 node::DLOpen(v8::FunctionCallbackInfo<v8::Value> const&) + 855
    5   node                                0x0000000100169e96 v8::internal::FunctionCallbackArguments::Call(void (*)(v8::FunctionCallbackInfo<v8::Value> const&)) + 378
    6   node                                0x00000001001aaeca v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) + 915
    7   node                                0x00000001001aa4a7 v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) + 281
    8   ???                                 0x00002388d58063a7 0x0 + 39070604485543
    9   ???                                 0x00002388d5c2d13a 0x0 + 39070608838970
Illegal instruction: 4
```

**Comment by retorquere** [comment]
The `npm i` went without issue or complaints though. This is on macos Sierra 10.12.2.

BTW `ncu` shows these updated packages, thought you might want to know:

```
 react-bootstrap    ^0.13.3  →  ^0.30.7
 browser-sync        ^1.8.1  →  ^2.18.6
 browserify          ^7.0.3  →  ^13.3.0
 gulp-autoprefixer   ^2.0.0  →   ^3.1.1
 gulp-less            2.0.1  →    3.3.0
 gulp-sourcemaps     ^1.2.8  →   ^2.4.0
 pretty-hrtime       ^0.2.2  →   ^1.0.3
 reactify           ^0.17.1  →   ^1.1.1
 require-dir         ^0.1.0  →   ^0.3.1
 watchify            ^2.2.1  →   ^3.8.0
```

**Comment by jogleasonjr** [comment]
Thanks for the tip about `ncu`, that's a handy utility. Today I reformatted my machine with macOS Sierra, installed NodeJS LTS, and cloned/built this project without seeing the error you specified. It's a pretty barebones machine right now. Is there something unique about your configuration? Are you using the LTS version of NodeJS?

I did use `ncu -u` for housekeeping and fixed any resulting warnings. No errors/warnings on my end. This is committed to master. Maybe it fixed some conflict that I wasn't seeing. Any better with the latest version?
